ouroboros-network/src/Ouroboros/Network/PeerSharing.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE NamedFieldPuns   #-}
+{-# LANGUAGE RecordWildCards  #-}
```

Comment:
Let's remove `RecordWildCards`; they hide where arguments are coming from and going to.

Don't we have it in our style guide?

Reply:
That's a good question, from a quick grep we use it on other modules

Reply:
I know, in a few places. I'd rather remove it than expand it.